Feature/inpro 2705 template for graphql yoga

loffler opened the pull request.

dusanscensny reviewed (approved):
Looks ok. I see on lint this warning: 
```
(node:18) [MODULE_TYPELESS_PACKAGE_JSON] Warning: Module type of file:///app/eslint.config.js?mtime=1747381825818 is not specified and it doesn't parse as CommonJS.
Reparsing as ES module because module syntax was detected. This incurs a performance overhead.
To eliminate this warning, add "type": "module" to /app/package.json.
(Use `node --trace-warnings ...` to show where the warning was created)
```
I would update in README that you need to run `nvm use` and `npm run dev` to run it locally. But looks ok

loffler commented:
@dusanscensny I fixed the eslint warning by renaming eslint.config.js to eslint.config.mjs and I also fixed few more issues.

For running the app locally, `$ task npm:run -- dev` should be used, as stated in README. This command automatically picks the correct version in .nvmrc.

But I updated README with the information that if you really want to use locally installed node.js, than you have to run `nvm use` manually.
